examples/perfect_catke_calibration.jl

```diff
@@ -51,13 +53,18 @@ data_path = generate_synthetic_observations("catke",
 
 # Next, we load and inspect the observations to make sure they're sensible:
 
-observations = SyntheticObservations(data_path, field_names=(:u, :v, :b, :e), transformation=ZScore())
+transformation = (u = Transformation(normalization=ZScore()),
+                  v = Transformation(normalization=ZScore()),
+                  b = Transformation(normalization=ZScore()),
+                  e = Transformation(normalization=RescaledZScore(1e-1)))
```

Comment:
Why not use the convenience interface? Should we delete the convenience interface?

Reply:
Basically if `transformation isa normalization` then we do the trivial wrappage under the hood. Seems like boiler plate otherwise (unless we end up doing transformations most cases, not sure. None of the examples do it).

Reply:
So that would be 
```
transformation = ZScore(),
                  v = ZScore(),
                  b = ZScore(),
                  e = RescaledZScore(1e-1))
```

Reply:
yas

Reply:
cause https://github.com/CliMA/OceanTurbulenceParameterEstimation.jl/blob/cebc56285f9c5df8d6e60b827455ffcdb50e2c02/src/Transformations.jl#L116-L120

Reply:
I'll fix it in #192